### ISSUE TITLE
Live demo needs to be updated

### ISSUE THREAD

**Issue by jameslyon** [typetools/checker-framework#976]
The following fails to compile (using the live demo):

```java
class YourClassNameHere {
    void foo(Enum<? extends CharSequence> nbl) {}
}
```

It fails with:

```
Error: [type.argument.type.incompatible] incompatible types in type argument.
  found : ? extends CharSequence
  required: Enum	
```

It works if replaced with:

```java
class YourClassNameHere {
    <T extends Enum<T> & CharSequence> void foo(T nbl) {}
}
```

This is valid Java, so should compile with the checker.

**Comment by smillst** [comment]
I can reproduce this problem using the Live Demo, but I cannot reproduce using the development version of the Checker Framework.  So, the Live Demo must not be up-to-date. We will do that. Thanks for reporting the issue!

**Comment by smillst** [comment]
The Live Demo is now up-to-date and your example works.
